orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/tasks/rollingpush/DetermineTerminationCandidatesTask.groovy

```diff
@@ -46,6 +46,9 @@ class DetermineTerminationCandidatesTask implements Task {
     def terminationInstancePool = knownInstanceIds
     if (stage.context.termination?.instances) {
       terminationInstancePool = knownInstanceIds.intersect(stage.context.termination?.instances)
+      if (stage.context.termination?.order == 'given') {
```

Comment:
minor but we don't need the `termination?.order` ... we've already guaranteed `termination` is non-null in this block.

same in the `.sort {}` below